### PR TITLE
CIty for users

### PR DESCRIPTION
**Pull request by aryehlev**
Users can add a default city on signup and put the city function so that it is in the right place

### CODE REVIEW

**Comment by OdedHellman** [comment]
Looks very good @aryehlev.
Good job!

**Comment by BenHurCreations** [comment]
very nice work! @aryehlev

File: superx/routes/home.py [hunk 1]
```diff
@@ -135,4 +134,21 @@ def remove_item():
...
 
+
+    return ''
+
+
+def city():
+    """
+    adds city that was chosen, using jquery to get the data and ajax so not to redirect
+    """
+    session['city'] = request.form.get('city')
```
Comment (simzacks) [inline superx/routes/home.py]: in pr #71  @OdedHellman and I discussed this functionality and moved it directly into routing.py.
My suggestion is to keep it that way.

File: superx/routing.py [hunk 2]
```diff
@@ -76,5 +76,11 @@ def city():
     """
     city route
     """
-    session['city'] = request.form.get("city")
-    return 'OK'
+    return home.city()
```
Comment (simzacks) [inline superx/routing.py]: see comment above

File: superx/routes/home.py [hunk 3]
```diff
@@ -13,8 +13,7 @@ def home():
     returns landing page of application
     """
 
-    city_list = db.session.query(Branch.city).order_by(Branch.city).distinct().all()
-    return render_template('home.html', city_list=city_list)
+    return render_template('home.html')
```
Comment (simzacks) [inline superx/routes/home.py]: in #71  the render_template call added the city_list parameter, which is used on line 21 of home.html.
You might want to keep it.

**Review by simzacks — CHANGES_REQUESTED** [review]
This PR overwrites some of the changes in PR #71 and doesn't look like it was intentional.

File: superx/routes/home.py (hunk 3 above)
Comment (aryehlev) [reply superx/routes/home.py]: yah, realized that afterwards that i need to give them the option if not signed in

File: superx/routes/signup.py [hunk 4]
```diff
@@ -85,6 +98,7 @@ def login():
         user = User.query.filter_by(email=form.email.data).first()
         if check_password_hash(user.password, form.password.data):
             login_user(user)
+            session['cart'] = current_user.city
```
Comment (simzacks) [inline superx/routes/signup.py]: should this be session['city'] instead of session['cart']?

**Review by simzacks — CHANGES_REQUESTED** [review]

File: superx/routes/signup.py (hunk 4 above)
Comment (aryehlev) [reply superx/routes/signup.py]: yah

Comment (aryehlev) [reply superx/routes/signup.py]: yah. my bad

**Review by simzacks — APPROVED** [review]

**Pull request merged**